pkg/skaffold/deploy/kubectl/kubectl.go

```diff
@@ -88,43 +89,62 @@ func (k *Deployer) Deploy(ctx context.Context, out io.Writer, builds []graph.Art
 		manifests manifest.ManifestList
 		err       error
 	)
+
+	ctx, endTrace := instrumentation.StartTrace(ctx, "Deploy")
```

Comment:
This should indicate the deployer type. WDYT:
```suggestion
	ctx, endTrace := instrumentation.StartTrace(ctx, "deploy_kubectl")
```

Or should it be an attribute?

Reply:
Done.  Added as an attribute